c/chosen/chosen_rhel_8_4.sh

```diff
@@ -0,0 +1,31 @@
+# -----------------------------------------------------------------------------
+#
+# Package       : chosen
+# Version       : 1.0.0
+# Source repo   : https://github.com/harvesthq/chosen/archive/1.0.0.zip
+# Tested on     : UBI 8
+# Script License: Apache License, Version 2 or later
+# Maintainer    : sachin.kakatkar@ibm.com
+#
```

Comment:
From now on wards, please start adding Language to the Script header.
E.g:
```
# Maintainer : sachin.kakatkar@ibm.com
# Language : Node
```

Reply:
Okay